### ISSUE TITLE
Memory Usage Increase Issue

### ISSUE THREAD

**Issue by pluto32** [lucidrains/CALM-pytorch#7]
When training, there is an issue where memory usage continuously increases during the loss calculation in the following part:

**loss = F.cross_entropy(
            rearrange(logits, 'b n c -> b c n'),
            labels,
            ignore_index=self.pad_id
        )**

I would like to ask if anyone else has experienced the same issue.
